chapters/ch44.md

```diff
@@ -1,85 +1,71 @@
 
---> _replace THIS LINE by your translation for the above line_
-
+2. **Vấn đề hàm cho điểm mục tiêu.** [cái này em chưa tìm được từ ngữ thích hợp để diễn tả] Ước tính của chúng tôi về Điểm​<sub>A​</sub>(​*S*) = P(​*S​*|​*A​*) không chính xác. Cụ thể, Điểm​<sub>A​</sub>(​*S*) chúng tôi chọn đã không thể nhận ra "Tôi yêu thích học máy" là chính xác.
 
 > Depending on which of these was the cause of the failure, you should prioritize your efforts
 > very differently. If #1 was the problem, you should work on improving the search algorithm.
 > If #2 was the problem, you should work on the learning algorithm that estimates Score​<sub>A​</sub>(​*S*).
 
---> _replace THIS LINE by your translation for the above line_
-
+Tùy thuộc vào nguyên nhân nào dẫn đến thất bại mà bạn ưu tiên tập trung vào hướng giải quyết khác nhau. Nếu #1 là vấn đề, bạn nên cải thiện thuật toán tìm kiếm. Nếu #2 là vấn đề, bạn nên làm việc với thuật toán học ước tính Điểm​<sub>A​</sub>(​*S*).
 
 > Facing this situation, some researchers will randomly decide to work on the search algorithm; others will randomly work on a better way to learn values for Score​<sub>A​</sub>(​*S*). But unless you know which of these is the underlying cause of the error, your efforts could be
 > wasted. How can you decide more systematically what to work on?
-
---> _replace THIS LINE by your translation for the above line_
-
+ 
+Đối mặt với tình huống này, một số nhà nghiên cứu ngẫu nhiên sẽ quyết định làm việc trên thuật toán tìm kiếm; những người khác sẽ ngẫu nhiên làm việc theo cách tốt hơn để tìm hiểu các giá trị cho Điểm​<sub>A​</sub>(​*S*). Nhưng nếu bạn không biết nguyên nhân nào trong số này là nguyên nhân cơ bản tạo nên lỗi, nỗ lực của bạn có thể trở nên lãng phí. Làm thế nào bạn có thể quyết định nên làm gì một cách có hệ thống hơn?
 
 > Let S​<sub>out</sub> be the output transcription (“I love robots”). Let S<sup>x</sup> be the correct transcription (“I love machine learning”). In order to understand whether #1 or #2 above is the problem, you can perform the ​**Optimization Verification test​**: First, compute Score​<sub>A​</sub>(​*S*<sup>x</sup>) and Score​<sub>A​</sub>(​*S*<sub>out</sub>). Then check whether Score​<sub>A​</sub>(​*S*<sup>x</sup>) > Score​<sub>A​</sub>(​*S*<sub>out</sub>). There are two possibilities:
 
---> _replace THIS LINE by your translation for the above line_
-
+Đặt S​<sub>out</sub> là đầu ra ("Tôi yêu thích người máy"). Đặt S<sup>x</sup> là bản chính xác ("Tôi yêu thích học máy"). Để hiểu vấn đề #1 hay # 2 ở trên là nguyên nhân, bạn có thể thực hiện **kiểm tra xác minh tối ưu hóa**: Đầu tiên, tính Điểm​<sub>A​</sub>(​*S*<sup>x</sup>) và Điểm​<sub>A​</sub>(​*S*<sub>out</sub>). Sau đó kiểm tra xem Điểm​<sub>A​</sub>(​*S*<sup>x</sup>) > Điểm​<sub>A​</sub>(​*S*<sub>out</sub>) không. Có hai khả năng:
```

Comment:
* [ ] Em chuyển hết "transcription" thành "bản phiên âm" nhé.
* [ ] Đoạn in đậm dịch như tiêu đề chương.
* [ ] Không có dấu cách giữa `#` và `2`.
* "Sau đó kiểm tra xem Điểm​<sub>A​</sub>(​*S*<sup>x</sup>) > Điểm​<sub>A​</sub>(​*S*<sub>out</sub>) _có đúng hay_ không."